tests/run-make/simd-ffi/rmake.rs

```diff
@@ -0,0 +1,63 @@
+// Using SIMD types in a program with foreign-function interfaces used to result in an ICE
+// (internal compiler error). Since this was fixed in #21233, it should be checked that
+// compilation of SIMD and FFI together should be successful on all the most common
+// architectures.
+// Note that this test does not check linking or binary execution.
+// See https://github.com/rust-lang/rust/pull/21233
+
+use run_make_support::{llvm_components_contain, rustc};
+
+fn main() {
+    let mut targets = Vec::new();
+    // arm-specific targets.
+    if llvm_components_contain("arm") {
+        targets.append(&mut vec![
+            "arm-linux-androideabi".to_owned(),
+            "arm-unknown-linux-gnueabihf".to_owned(),
+            "arm-unknown-linux-gnueabi".to_owned(),
+        ]);
```

Comment:
Nit: this can be `targets.append(&[])` (ref-to-array) right?

Reply:
I don't think so:

```
     = note: expected mutable reference `&mut Vec<_>`
                found reference `&[std::string::String; 3]`
```

I've tried various permutations of this, it seems to accept nothing except what I currently have. There might be some clever work-around I'm not aware of, but I will r= the PR for now. Feel free to stop it or invite a revision in a further PR if I misunderstood something.